vzstorage-pd/virtuozzo-provisioner.go

```diff
@@ -47,6 +48,10 @@ const (
 	failedRetryThreshold      = 5
 	parentProvisionerAnn      = "vzFSParentProvisioner"
 	vzShareAnn                = "vzShare"
+	leasePeriod               = leaderelection.DefaultLeaseDuration
+	retryPeriod               = leaderelection.DefaultRetryPeriod
+	renewDeadline             = leaderelection.DefaultRenewDeadline
+	termLimit                 = leaderelection.DefaultTermLimit
```

Comment:
Currently you don't need these constants, if you don't want to change default values.
https://github.com/kubernetes-incubator/external-storage/blob/master/lib/controller/controller.go#L317